hdfs_namenode/datadog_checks/hdfs_namenode/hdfs_namenode.py

```diff
@@ -26,6 +26,9 @@ class HDFSNameNode(AgentCheck):
     # Namesystem bean
     HDFS_NAME_SYSTEM_BEAN = 'Hadoop:service=NameNode,name=FSNamesystem'
 
+    # Namesystem info bean
+    HDFS_NAME_SYSTEM_METADATA_BEAN = 'Hadoop:service=NameNode,name=NameNodeInfo'
```

Comment:
Mind if we do this one in a separate PR?

Reply:
Cherry picked into separate branch, and force updated this one.